modules/EasyAbp.EShop.Products/src/EasyAbp.EShop.Products.Application/EasyAbp/EShop/Products/ProductCategories/ProductCategoryAppService.cs

```diff
@@ -34,6 +34,8 @@ protected override async Task<IQueryable<ProductCategory>> CreateFilteredQueryAs
                 queryable = queryable.Where(x => x.ProductId == input.ProductId);
             }
 
+            queryable = queryable.OrderBy(x => x.DisplayOrder);
```

Comment:
It is better to sort in `ApplySorting`. Is it expected to always sort by CreationTime when `Input.Sorting` is empty?

https://github.com/abpframework/abp/blob/edda93473dd458af31be82ec55cd371ec2a952ce/framework/src/Volo.Abp.Ddd.Application/Volo/Abp/Application/Services/AbstractKeyReadOnlyAppService.cs?#L93-L131

Reply:
I Think the sequential is enough. CreationTime sorting needs additional indexes.